frontend/localConfig.json

```diff
@@ -1,5 +1,5 @@
 {
-    "geoStoreUrl": "/sira/rest/geostore",
+    "geoStoreUrl": "/decsiraweb/rest/geostore",
```

Comment:
You can get the same result with a relative path:

"geoStoreUrl": "rest/geostore",